doc recordfinder dot notation for namefrom (#372)
octobercms/docs: backend-forms.md

Option | Description
------------- | -------------
**keyFrom** | the name of column to use in the relation used for key. Default: id.

> **nameFrom** | the column name to use in the relation used for displaying the name. Can use the dot notation to query a related model (e.g. use name from student model: student.name). It can use unlimited nesting such as relation.subrelation.subsubrelation.propertythatreturnsanobject.name. Default: name.

Reviewer: ```suggestion
**nameFrom** | the column name to use in the relation used for displaying the name. You may use dot notation to query a related model (e.g. to use the name field from student model: `student.name`). It can use unlimited nesting, such as `relation.subrelation.subsubrelation.name`. Default: name.
```
